test_pull_requests

```diff
@@ -124,12 +124,38 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_RUNNING = "Running"
+CONTENT_WAITING = "Waiting"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
```

Comment:
good catch. The code will work without it, but it is one of the possible results of `JenkinsAPI::get_build_result` so may as well add it.